sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs

```diff
@@ -143,12 +191,44 @@ public async Task SendingSmsMessageToGroupWithOptions()
                 foreach (SmsSendResult result in results)
                 {
                     Console.WriteLine($"Sms id: {result.MessageId}");
-                    /*@@*/ Assert.IsFalse(string.IsNullOrWhiteSpace(result.MessageId));
-                    /*@@*/ Assert.AreEqual(202, result.HttpStatusCode);
-                    /*@@*/ Assert.IsTrue(result.Successful);
+                    Assert.IsFalse(string.IsNullOrWhiteSpace(result.MessageId));
+                    Assert.AreEqual(202, result.HttpStatusCode);
+                    Assert.IsTrue(result.Successful);
                 }
-                #endregion Snippet:Azure_Communication_SmsClient_Send_GroupSmsWithOptions
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+        }
+
+        [Test]
+        public async Task SendingTwoSmsMessages()
+        {
+            SmsClient client = InstrumentClient(
+                new SmsClient(
+                    TestEnvironment.ConnectionString,
+                    InstrumentClientOptions(new SmsClientOptions())));
+            try
+            {
+                SmsSendResult firstMessageResult = await client.SendAsync(
+                   from: TestEnvironment.FromPhoneNumber,
+                   to: TestEnvironment.ToPhoneNumber,
+                   message: "Hi");
+                SmsSendResult secondMessageResult = await client.SendAsync(
+                   from: TestEnvironment.FromPhoneNumber,
+                   to: TestEnvironment.ToPhoneNumber,
+                   message: "Hi");
 
+                Assert.AreNotEqual(firstMessageResult.MessageId, secondMessageResult.MessageId);
+                Assert.True(firstMessageResult.Successful);
```

Comment:
Did you consider having a helper method to verify successful messages? You can also use it for the other happy case tests.

Reply:
Great suggestion! Thanks